src/kvstore/raftex/Host.cpp

```diff
@@ -190,6 +190,10 @@ void Host::appendLogsInternal(folly::EventBase* eb,
             {
                 std::lock_guard<std::mutex> g(self->lock_);
                 self->setResponse(r);
+                self->lastLogIdSent_++;
```

Comment:
why NOT  self->lastLogIdSent_ =  self->logIdToSend_

Reply:
Resonable!